src/xdocs/config_misc.xml

```diff
@@ -1084,6 +1084,23 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
           times it is practical experience. In any case, this check should
           just ensure that a minimal set of indentation rules is followed.
         </p>
+        <p>
+          Basic offset indentation is used for indentation inside code blocks.
+          Line wrapping indentation is used when statement or expression is placed on
+          few lines.
```

Comment:
I would expand documentation to say what the priority is. Line wrapping will always take over basic offset if the line is being wrapped for example.

Reply:
Please suggest exact wording, you are most experienced in this Check, I do not know all details.

Reply:
Done.